.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.6', '1']
+        version: ['1.3', '1']  # Test against current minor release (at least 1.3)
```

Comment:
this is an error in the merge, should remain as is

Reply:
I've changed it back to 1.6

Reply:
it still appears here?

Reply:
I made the same mistake in MINLP.yl and reverted it there but not here. Should be back to 1.6 in both now.